pkgs/by-name/v4/v4l2-to-ndi/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  openssl,
+  curl,
+  avahi,
+  ndi,
+}:
+
+stdenv.mkDerivation {
+  pname = "v4l2-to-ndi";
+  version = "unstable-2022-09-14";
```

```suggestion
  version = "0-unstable-2022-09-14";
```
I missed this last time.